tdrs-backend/tdpservice/data_files/static/admin/js/admin/admin_datafile_model.js

```diff
@@ -2,29 +2,47 @@ $(window).on('load', function() {
     console.log('loaded');
     var submitBtn=document.querySelector('button[type=submit]');    // add the first listener
     var theForm = submitBtn.parentNode.parentNode;
+    var action = "";
+    var number_of_files_line = "";
 
-    for (var i = 0; i < theForm.childNodes.length; i++) {
-        if (theForm.childNodes[i].className == "actions") {
-          form_header = theForm.childNodes[i];
-
-          for (var i = 0; i < form_header.childNodes.length; i++) {
-              if (form_header.childNodes[i].className == "action-counter") {
-                number_of_files = form_header.childNodes[i];
-                break;
-              }
-            }
-
-          break;
-        }
-    }
     submitBtn.addEventListener('click', function(e) {
         e.preventDefault();
-        if (confirm("You are about to re-parse " + number_of_files.innerHTML.split(/(\s+)/)[0] + " files. Are you sure you want to continue?")) {
-            console.log('submitting');
-            theForm.submit();
+        for (var i = 0; i < theForm.childNodes.length; i++) {
```

Comment:
This works as advertised. But is there not a way for us to select the element containing the selected number of files instead of looping through children manually? Could we do something more like: `document.querySelector("span.all");`? I tried in the dev tools to see if it works and it returns the right element. We would just need to split on the space character to extract the number. We would also need to check that the element isn't hidden. If it was, then we just grab the other span.

Reply:
Forgot to include the screenshot:

![Screenshot 2024-11-14 at 2 16 42 PM](https://github.com/user-attachments/assets/f8062c54-4977-40a7-8236-85531534a1a1)